vscode-car-plugin/pom.xml

```diff
@@ -88,9 +88,15 @@
             <artifactId>maven-invoker</artifactId>
             <version>${org.apache.maven.shared.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <version>${snakeyaml.version}</version>
+        </dependency>
     </dependencies>
     <properties>
         <org.apache.ws.commons.axiom.version>1.2.20</org.apache.ws.commons.axiom.version>
         <org.apache.maven.shared.version>3.1.0</org.apache.maven.shared.version>
+        <snakeyaml.version>1.33</snakeyaml.version>
```

Comment:
vulnerable snakeyaml version

Reply:
We are unable to move to the latest snakeyaml version because it can only be built in java 11 and this repo uses java 8. Also this vulnerability cannot be exposed because the yml files are provided by us via connectors which cannot contain vulnerable components. Also, this jar does not get packed to the runtime or VSCode extension.